mars/services/scheduling/worker/quota.py

```diff
@@ -269,29 +272,37 @@ def __init__(self, band: BandType, quota_size: int, hard_limit: int = None,
         self._refresh_time = refresh_time or 1
 
         self._stat_refresh_task = None
+        self._slot_manager_ref = None
 
     async def __post_create__(self):
         await super().__post_create__()
         self._stat_refresh_task = self.ref().update_mem_stats.tell_delay(delay=self._refresh_time)
 
+        from .workerslot import BandSlotManagerActor
```

Comment:
Move it to beginning of the function or file.

Reply:
Close to the user location, no need to move.